source/P0323R9.bs

```diff
@@ -941,16 +946,16 @@ otherwise
 
 *Throws*: Any exception thrown by the operations specified in the effect clause.
 
-*Remarks*: If an exception is thrown during the call to `T`'s assignment, nothing
-changes.
+*Remarks*: If an exception is thrown during the call to `T`'s assignment, `*this` remains
+unchanged.
 
 
 ```c++
 template<class U, class... Args>
     T& emplace(initializer_list<U> il, Args&&... args);
 ```
 
-*Constraints*: `T` is not *cv* `void` and `is_nothrow_constructible_v<T, initializer_list<U>&, Args...>` is `true`.
+*Constraints*: `T` is not *cv* `void` and `is_constructible_v<T, initializer_list<U>, Args...>` is `true` and (`is_nothrow_constructible_v<T, initializer_list<U>, Args...>` is `true` or `is_nothrow_move_constructible_v<T>` is `true` or `is_nothrow_move_constructible_v<E>` is `true`).
```

Comment:
The three ORed conditions within parentheses correspond to the three "otherwise" clauses under Effects.